router/state.go

```diff
@@ -267,3 +267,29 @@ func (s *state) _portDisconnected(peer *peer) {
 		s._bootstrapNow()
 	}
 }
+
+func (s *state) _lookupPeer(addr net.Addr) *peer {
```

Comment:
Can you please comment that this function is trying to find the peer for a given address, or name it as such? `_lookupPeerForAddr` or similar.